values/jobs/harbor.gotmpl

```diff
@@ -31,6 +32,7 @@ tasks:
       repository: {{ $c | get "jobs.harbor.image.repository" "otomi/tasks" }}
       tag: {{ $c | get "jobs.harbor.image.tag" "v0.2.1" }}
       pullPolicy: {{ $c | get "jobs.harbor.image.pullPolicy" "IfNotPresent" }}
+    name: harbor
```

Comment:
why is this redundant property put there? It is implicitly defined as key in line 18.

Reply:
I think this is a bit contradictory if we change it to an array of jobs? But yes it was definitely redundant.